controllers/datadogagent/component/clusteragent/default.go

```diff
@@ -144,6 +146,18 @@ func defaultEnvVars(dda metav1.Object) []corev1.EnvVar {
 			Name:  apicommon.DDHealthPort,
 			Value: strconv.Itoa(int(apicommon.DefaultAgentHealthPort)),
 		},
+		{
+			Name:  apicommon.DDInstrumentationInstallId,
+			Value: component.AgentInstallId,
+		},
+		{
+			Name:  apicommon.DDInstrumentationInstallTime,
+			Value: component.AgentInstallTime,
+		},
+		{
+			Name:  apicommon.DDInstrumentationInstallType,
+			Value: component.DefaultAgentInstallType,
+		},
```

Comment:
Are these env vars not specific to the APM use case?

Reply:
Yes, they power APM Onboarding KPIs. Should we add _APM_ prefix to the env variable names?

Reply:
no need if the instrumentation metadata naming has a consensus across product (wherever that could be used). However, the pattern we use is to only add feature specific configs when the config is enabled.  In this case, APM and the cluster agent are not yet tied (will be done in SSI), so while it'd be best to add more logic, this should be OK since we want APM to enabled.